### PR TITLE
[BXMSDOC-6453] Setting LDAP failover server

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc [hunk 1]
```diff
@@ -31,6 +31,12 @@ If you selected `LDAP`, configure LDAP authentication:
 
 .. In the *LDAP URL* field, enter the LDAP URL.
 .. Configure LDAP parameters that correspond to the settings of the LdapExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/login_module_reference/login_modules_with_external_identity_store#ldapextended_login_module[LdapExtended Login Module].
++
+[NOTE]
+====
+If you want to enable failover to another LDAP server, you can set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter, separated by the space character.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc:37]: I would change this 
```
If you want to enable failover to another LDAP server
```

to 

```
If you want to enable LDAP failover, you ca...
```

**Review by spolti — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc (hunk 1 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc]: I checked and the term is widely used, so I made the update.

**Review by spolti — APPROVED** [review]

File: doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc [hunk 2]
```diff
@@ -31,6 +31,12 @@ If you selected `LDAP`, configure LDAP authentication:
 
 .. In the *LDAP URL* field, enter the LDAP URL.
 .. Configure LDAP parameters that correspond to the settings of the LdapExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/login_module_reference/login_modules_with_external_identity_store#ldapextended_login_module[LdapExtended Login Module].
++
+[NOTE]
+====
+If you want to enable LDAP failover, you can set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter, separated by the space character.
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc:37]: Minor suggestion - Change article of space and remove "character". IBMSG appears to refer to the space character as "space" or "a space". I tried rewriting the note, please disregard if it doesn't convey the information correctly.

s/If you want to enable LDAP failover, you can set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter, separated by the space character./To enable LDAP failover, set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter with each address separated by a space.

File: doc-content/enterprise-only/openshift/template-deploy-ldap-proc.adoc [hunk 3]
```diff
@@ -18,6 +18,11 @@ You must create a user with the username and password configured in the secret f
 .Procedure
 . Set the `AUTH_LDAP*` parameters of the template. These parameters correspond to the settings of the `LdapExtended` Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/login_module_reference/#ldapextended_login_module[LdapExtended login module].
 +
+[NOTE]
```
Comment (csherrar) [inline doc-content/enterprise-only/openshift/template-deploy-ldap-proc.adoc:21]: Minor suggestion - Change article of space and remove "character". IBMSG appears to refer to the space character as "space" or "a space". I tried rewriting the note, please disregard if it doesn't convey the information correctly.

s/If you want to enable LDAP failover, you can set two or more LDAP server addresses in the AUTH_LDAP_URL parameter, separated by the space character./To enable LDAP failover, set two or more LDAP server addresses in the AUTH_LDAP_URL parameter with each address separated by a space.

**Pull request merged**